sdk/iot/provisioning/inc/az_iot_provisioning_client.h

```diff
@@ -63,7 +65,8 @@ AZ_NODISCARD az_iot_provisioning_client_options az_iot_provisioning_client_optio
  * @param[in] id_scope The ID Scope.
  * @param[in] registration_id The Registration ID. This must match the client certificate name (CN
  *                            part of the certificate subject).
- * @param[in] options A reference to an #az_iot_provisioning_client_options structure. Can be NULL.
+ * @param[in] options __[nullable]__ A reference to an 
+ *                                   #az_iot_provisioning_client_options structure. Can be NULL.

```

Comment:
Can be NULL for default options.

Reply:
Also, add back ticks around language keywords like NULL, true, false.